Subscribe to pending events on startup

benthecarman opened the pull request:
Closes #171

futurepaul commented:
I got notified of my two onchain txs on startup which was cool. but I did a receive, closed the app, got notified but it doesn't say pending even though there's only 2 confs

<img width="826" alt="Screenshot 2025-02-07 at 3 44 30 PM" src="https://github.com/user-attachments/assets/2e6de3bc-e563-45bb-a004-d7f3a20a9611" />

futurepaul reviewed (approved):
looks straightforward and already improves things. can keep tweaking how we show pending going forward

benthecarman commented:
Wasn't saying pending because it wasn't on top of that PR, rebased so now it should. Also fixed it so it won't resend the notifications for txs that we've already seen and are just waiting for confirmation
